docs/knowledge_base/demos/image_classif.md

```diff
@@ -0,0 +1,47 @@
+Welcome to our video tutorial series, where we take you on a journey to create interactive image 
+classification applications using Taipy, [Nvidia CUDA](https://developer.nvidia.com/cuda-toolkit)
+, and [TensorFlow](https://www.tensorflow.org/). In this demonstration, we'll dive into the 
+fascinating realm of constructing intelligent image classifiers using these robust tools and 
+technologies.
+
+
+# Video 1: Building an Interactive Image Classification Application with Taipy and Nvidia CUDA
+
+In the initial video of our series, we will demonstrate how to construct an interactive image 
+classification application using Taipy and Nvidia CUDA. You will swiftly grasp the process of 
+building an Image Classification web application with the user-friendly Taipy front-end.
+
+## Video Highlights:
+
+Wrap a Deep Learning Image Classifier with interactive graphics.
+Utilize Nvidia CUDA for enhanced performance.
+Explore the beginner/intermediate-level tutorial on Taipy GUI.
+Part 2 Preview:
+
+Don't miss out on part 2, where we delve into creating powerful custom pipelines with Taipy Core,
+taking your ML workflows to the next level.
+
+
+# Video 2: Interactive Image Classification with Taipy 
+
+In our second video, we continue our journey through interactive image classification 
+applications with Taipy. This installment is dedicated to constructing an Image Classification 
+web application, building upon the insights gained from the first video.
+
+## Video Highlights:
+
+Further explore Taipy's capabilities for ML application development.
+Dive into creating an interactive Image Classification web application.
+
+# Conclusion
+
+Our video tutorial series provides valuable insights into the realm of interactive image 
+classification using Taipy, Nvidia CUDA, and TensorFlow. Whether you're a novice or a seasoned 
+developer, you'll discover practical guidance and real-world examples to enhance your machine 
+learning workflows.
+
+Stay tuned for the next part of our series, where we'll focus on the management of data 
+pipelines using Taipy Core, ensuring smooth integration with Taipy for enhanced functionality.
```

Comment:
1. The tutorials are not grouped by format (video) anymore. So I would remove this part.
2. If you don't want to remove it, we can talk about Taipy directly. No need to use the term Core.